src/styles/default/pretty.jl

```diff
@@ -320,7 +319,7 @@ function format_docstring(style::AbstractStyle, state::State, text::AbstractStri
                     write(deindented, line)
                 else
                     write(deindented, '\n')
-                    write(deindented, SubString(line, user_indent + 1, length(line)))
+                    write(deindented, chop(line; head = user_indent, tail = 0))
```

Comment:
It seemed a bit easier to use `chop` here (which returns a `SubString`) but if preferred this could also be written using `SubString`:
```suggestion
                    write(deindented, SubString(line, nextind(line, firstindex(line), user_indent), lastindex(line))
```